newIDE/app/src/UI/List.js

```diff
@@ -127,7 +127,7 @@ export class ListItem extends React.Component<ListItemProps, ListItemState> {
           <ElementWithMenu
             ref={elementWithMenu => (this._elementWithMenu = elementWithMenu)}
             element={
-              <IconButton edge="end" aria-label="menu">
+              <IconButton size="small" edge="end" aria-label="menu">
```

Comment:
This was the mistake, by the way, the `IconButton` had a large margin, which made the scrollbar appear :sweat_smile: With a small size, everything is fine :+1: